src/test/java/net/datafaker/formats/XmlTest.java

```diff
@@ -21,18 +21,15 @@ void xmlPrettyTest(Xml.XmlNode xmlNode, String expected) {
     }
 
     private static Stream<Arguments> generateTestXmlPretty() {
-        return Stream.of(
-            Arguments.of(new Xml.XmlNode("root", Collections.emptyList()), "<root/>"),
-            Arguments.of(new Xml.XmlNode("root", "value"), "<root>value</root>"),
-            Arguments.of(new Xml.XmlNode("root",
-                    map(entry("attribute1", "value1"), entry("attribute2", "value2")), "value"),
-                "<root attribute1=\"value1\" attribute2=\"value2\">value</root>"),
-            Arguments.of(new Xml.XmlNode("root",
-                    map(entry("attribute1", "value1"), entry("attribute2", "value2")),
-                    Collections.singletonList(new Xml.XmlNode("child", "value"))),
-                "<root attribute1=\"value1\" attribute2=\"value2\">" + System.lineSeparator() + "    <child>value</child>" + System.lineSeparator() + "</root>"),
-            Arguments.of(new Xml.XmlNode("root", "<> value\""), "<root>&lt;&gt; value&quot;</root>")
-        );
+        return Stream.of(Arguments.of(new Xml.XmlNode("root", Collections.emptyList()), "<root/>"), Arguments.of(new Xml.XmlNode("root", "value"), "<root>value</root>"), Arguments.of(new Xml.XmlNode(
```

Comment:
Any chance to get this into a somewhat more reasonable state? I think the previous formatting was quite a bit better?

Reply:
should be better now